integration-tests/konflux_test_validation.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: konflux-test-validation
+spec:
+  description: |
+    The konflux-test validation integration pipeline runs the tests within the newly built konflux-test image. 
+    This verifies that the OPA policies and the bash functions that were built into the image
+    conform to the defined testing requirements.
+  params:
+    - description: |
+        Spec section of an Snapshot resource. Not all fields of the
+        resource are required. A minimal example:
+          {
+            "components": [
+              {
+                "containerImage": "quay.io/example/repo:latest",
+                "source": {
+                  "git": {
+                    "url": "https://github.com/org/repo",
+                    "revision": "6c1cbf4193930582d998770411f81d5c70aea29b"
+                  }
+                }
+              }
+            ]
+          }
+      name: SNAPSHOT
+      type: string
+  tasks:
+    - name: test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/tekton-integration-catalog
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: /tasks/test-metadata/0.1/test-metadata.yaml
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: test-name
+          value: "konflux-test"
+    - name: shellcheck
+      runAfter:
+        - test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/tekton-integration-catalog.git
```

Comment:
I see same declaration on line 35, without **.git**, are both declarations valid?

Reply:
Yes, Tekton supports both formats, but I'll update the refs for consistency.

Reply:
Updated so both now use .git